doc/build.rst

```diff
@@ -160,9 +159,9 @@ The Python package is located at ``python-package/``.
 
 1. Install system-wide, which requires root permission:
 
-.. code-block:: bash
+   .. code-block:: bash
 
-  cd python-package; sudo python3 setup.py install
+     cd python-package; sudo python3 setup.py install
```

Comment:
This may also be indented two spaces too many.

Reply:
Same as above.